lib/printer.js

```diff
@@ -2583,8 +2583,7 @@ function printExportDeclaration(path, options, print) {
     if (decl.declaration) {
         parts.push(path.call(print, "declaration"));
 
-    } else if (decl.specifiers &&
-               decl.specifiers.length > 0) {
+    } else if (decl.specifiers) {
```

Comment:
This change makes sense, but can we also make `shouldPrintSpaces` false if `decl.specifiers.length === 0`, to avoid the extra spaces between the braces?